Bottom: 149a333bd2b79ef125e96607d91fa8f10b9155e8
Top:    0a9005e21ff77f266c98c49bb8473c3d295a6c35
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-12-10 15:48:09 -0800

Refresh of dsocket

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 56f6f39..7dc5409 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1040,7 +1040,6 @@ int rsocket(int domain, int type, int protocol)
 		rs->cm_id->route.addr.src_addr.sa_family = domain;
 		index = rs->cm_id->channel->fd;
 	} else {
-printf("rsocket DGRAM\n");
 		ret = ds_init(rs, domain);
 		if (ret)
 			goto err;
@@ -1887,6 +1886,9 @@ static void ds_poll_cqs(struct rsocket *rs)
 	int ret, cnt;
 
 	qp = rs->qp_list;
+	if (!qp)
+		return;
+
 	do {
 		cnt = 0;
 		do {
